transform_spans.go

```diff
@@ -38,10 +38,14 @@ func ocSpanToProtoSpan(sd *trace.SpanData) *tracepb.Span {
 	if sd.Name != "" {
 		namePtr = &tracepb.TruncatableString{Value: sd.Name}
 	}
+	var parentSpanID []byte
+	if sd.ParentSpanID != (trace.SpanID{}) {
```

Comment:
Not sure why this needs to be fixed here and not in the Jaeger gRPC exporter in the collector.